pysrc/SBBox.cpp

```diff
@@ -35,36 +35,40 @@ namespace galsim {
     struct PySBBox 
     {
 
-        static SBBox* construct(const bp::object & width, const bp::object & height, double flux,
```

Comment:
Right.  The `bp::init` syntax is simpler for classes that don't need to do anything special with the arguments (like we do with profiles that can take multiple kinds of sizes).  So this hand-made initializer was basically doing what boost does for us with `bp::init` (i.e. check for the presence of required arguments).